tests/modules/nf-core/mitohifi/mitohifi/nextflow.config

```diff
@@ -1,5 +1,7 @@
 process {
-
+    ext.args2 = '-c'
```

Comment:
Can you do a test with -r too or is that too complicated?

Reply:
Yes, we can do it. Thank you for suggestion.